src/Service/PaymentService.php

```diff
@@ -117,25 +118,23 @@ public function checkPayment($id)
                 if ($paymentResponsebody['source']['type'] === 'sofort' && $paymentResponsebody['status'] === 'Pending' || $paymentResponsebody['status'] == 'Captured') {
                     $response['state'] = self::PAYMENT_APPROVED;
                 }
-                
-                
             } else {
                 $response['state'] = self::PAYMENT_ERROR;
-                $response['message'] = "An error has occured"; // @todo give proper error message to frontstore
+                $response['message'] = "An error has occured";
             }
 
             return $response;
             
         } catch (Exception $e) {
-            // @todo catch and log error
 
+            CkoLogger::log($e->getMessage(), "cko verify payment", "checkout.payment.verify.error", $id, "Error");
```

Comment:
Can we directly reference the logger instance, so we can bypass the need to map to a loglevel, and just direct log a message, for example:

```
CkoLogger::loggger->Error(array(
    "type": => "checkout.create.payment.error"
    "scope" => "cko create payment",
    "message" => $e->getMessage()
))
```